servicetalk-grpc-internal/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2019, 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
```

Comment:
Don't we need the `concurrent-api` and `http-api` to be `api` dependencies? They are part of the public API.

Reply:
Using them as implementation here doesn't affect their publication in other modules and this module only needs them as implementation. ie. it doesn't export any types which are defined in either `concurrent-api` or `http-api`.

Reply:
`http-api` does need to be an `api` dependency.